client_android/app/src/pro/java/com/securecall/app/ads/AdMobManager.kt

```
@@ -7,6 +7,9 @@ import android.widget.FrameLayout
 /** No-op AdMob stub for PRO flavor — no ads. */
 object AdMobManager {
     var isInitialized = false; private set
+    fun requestConsentAndLoad(activity: Activity, container: FrameLayout) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)